### PR TITLE
merge release/5.4 into develop after code freeze

### PR DESCRIPTION
**Pull request by AliSoftware**
Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by AliSoftware** [comment]
Does the Peril warning above means I missed a step during my release scenario?

File: .idea/compiler.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
       <entry name="!?*.kt" />
       <entry name="!?*.clj" />
     </wildcardResourcePatterns>
-    <bytecodeTargetLevel>
+    <bytecodeTargetLevel target="11">
```
Comment (loremattei) [inline .idea/compiler.xml]: Why this change?

Comment (AliSoftware) [reply .idea/compiler.xml]: Good question. That's something that Android Studio did on its own when I built the project right after updating fluxC (so that jitpack would create the first build) I think. I can revert – and we can check if that comes back later (maybe a new thing introduced by AS 4.1 or something?)

Comment (AliSoftware) [reply .idea/compiler.xml]:
```suggestion
    <bytecodeTargetLevel>
```

**Comment by AliSoftware** [comment]
Note: updated the release notes (with ones from p2JRYi-56w-p2) since your last review

**Review by jkmassel — APPROVED** [review]

**Review by loremattei — APPROVED** [review]
